Client/Client.Qt/Widgets/LoadingSpinner.hpp

```diff
@@ -47,10 +47,9 @@ private slots:
     /// Put on center of page
     void updatePosition();
 
-private:
     const Style::LoadingSpinner& _st;
     std::unique_ptr<QTimer>      _timer;
     bool                         _disableParentWhenSpinning;
-    int32_t                      _currentCounter;
-    bool                         _isSpinning;
+    int32_t                      _currentCounter = 0;
```

Comment:
I think so there don't use int32_t. For this reason enough uint16_t.

![image](https://user-images.githubusercontent.com/70116007/188473811-1cb4ee69-bfc6-4f0a-9149-c2057aa47b05.png)